consultation_analyser/support_console/views/consultations.py

```diff
@@ -63,7 +63,23 @@ def get_number_themes_for_processing_run(processing_run):
 @staff_member_required
 def show(request: HttpRequest, consultation_id: UUID) -> HttpResponse:
     consultation = models.Consultation.objects.get(id=consultation_id)
+    print(request.POST)
     try:
+        download_key = None
+        for key in request.POST:
```

Comment:
Does this need to be a `POST`? Would it make more sense to have the button be a plain `<a>` pointing to e.g. `consultation/{slug}/processing_runs/{slug}.json` and then have that as a separate view serving just the file?

Reply:
Doesn't need to be a post - maybe that would be better (and look neater too). Will update.

Reply:
Yep - have changed this view back, and have a separate view for the downloads for each processing run.